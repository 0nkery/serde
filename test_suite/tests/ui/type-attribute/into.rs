#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(into = "Option<T")]
enum TestOne {
    Testing,
    One,
    Two,
    Three,
}
