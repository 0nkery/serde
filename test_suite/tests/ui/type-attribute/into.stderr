error: failed to parse type: into = "Option<T"
  --> $DIR/into.rs:12:10
   |
12 | #[derive(Serialize)]
   |          ^^^^^^^^^

error: aborting due to previous error

